legislators-social-media.yaml

```diff
@@ -5823,6 +5826,7 @@
     govtrack: 400343
     thomas: '00979'
   social:
+    twitter: DanaRohrabacher
```

Comment:
His [official website](https://rohrabacher.house.gov/) doesn't link to this Twitter account, and the account links to a Facebook page. I would not add this, unless you called his legislative office directly to verify.